Sources/DarkModeCore/DMTraitCollection.h

```diff
@@ -33,4 +42,10 @@ typedef NS_ENUM(NSInteger, DMUserInterfaceStyle) {
 
 @end
 
+@interface NSObject (DMTraitEnvironment)
+
++ (void)swizzleTraitCollectionDidChangeToDMTraitCollectionDidChange API_AVAILABLE(ios(13.0));
+
+@end
+
```

Comment:
This is a public header of the framework, right? We should think carefully about what to put here.

Reply:
In SPM, we cannot mix Objective-C and Swift in one target, this is a function that we needed to call in Swift, it has to be exposed to Swift, which in turn has to be exposed to a project that uses this framework. We might be able to solve it when SPM supports mixing Objective-C and Swift in one target, or move the codebase entirely to Objective-C

Reply:
I think this probably can be achieved by defining a internal module using modulemap. Have you tried this method before?

Reply:
let's leave it to be solved in another PR, it is an existing issue.